### PR TITLE
fixed file size bug

### PR DESCRIPTION
**Pull request by comountainclimber**
https://github.com/CityOfZion/neon-wallet-website/issues/7

screen shot 2018-04-30 at 5 32 05 pm

### CODE REVIEW

**Comment by mhuggins** [comment]
From the screenshot, it looks like the Mac OS download is the same as Windows (exe instead of dmg, and the same size).

File: src/config.js [hunk 1]
```diff
@@ -9,31 +9,35 @@ export const downloadOptions = [
...
   },
   {
     id: "MAC_OS",
     display: "Mac OS",
     href:
       "https://github.com/CityOfZion/neon-wallet/releases/download/0.1.1/Neon-0.1.1.Mac.dmg",
     icon: appleIcon,
-    size: "134.56 MB (Neon.dmg)"
+    size: "134.56 MB (Neon.dmg)",
+    fileExtension: "exe"
```
Comment (comountainclimber) [inline src/config.js]: @mhuggins thanks!

**Comment by comountainclimber** [comment]
yep @mhuggins 🙈 thanks for taking a look really quick lol

**Comment by warwickchapman** [comment]
Sweet.

**Pull request merged**